examples/topics/batched_example/GSSHA_Workflow_Batched_Example1.ipynb

```diff
@@ -0,0 +1,736 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Batched example 1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is the first of a series that shows how [GSSHA_Workflow.ipynb](../GSSHA_Workflow.ipynb) can be parameterized at the command line. This parameterization is designed to enable parameter sweeps, automatic report generation and other new uses for notebooks.\n",
+    "\n",
+    "This first notebook aims to explain the basic mechanism used to parameterize notebooks. Although this is the least user friendly of the approaches, it also requires the fewest changes to EarthSim and the associated PyViz tools.\n",
+    "\n",
+    "The rest of the notebook is largely the same as [GSSHA_Workflow.ipynb](../GSSHA_Workflow.ipynb) although there are a few changes that are used to expose the notebook parameters at the command line. The two parameters we choose to expose are ``rain_intensity`` and ``rain_duration``.\n",
+    "\n",
+    "\n",
+    "1. <a href=\"#Declare_nbparams\">Declare the command line parameters</a>\n",
+    "2. <a href=\"#Display_nbparams\">Display the notebook parameter widgets</a>\n",
+    "3. <a href=\"#Apply_nbparams\">Apply notebook parameters and display</a>\n",
+    "\n",
+    "\n",
+    "Once the notebook is configured to use notebook parameters, they can be set at the command line as follows:\n",
+    "\n",
+    "```bash\n",
+    "cd examples/topics/batched_example/\n",
+    "PARAM_JSON_INIT='{\"rain_intensity\":42, \"rain_duration\":50}' jupyter notebook GSSHA_Workflow_Batched_Example1.ipynb\n",
+    "```\n",
```

Comment:
Can you add the example nbconvert command as well?

Reply:
Addressed in 49912a8. It is a long command!